### PR TITLE
deps: update x/tools and gopls to d1954cc86c82

### PR DESCRIPTION
**Pull request by leitzler**
Note that this change also shorten the default errlogmatch wait time
from 30s to 25s to slightly improve CI test time and avoid hitting the
10 minute limit.

* internal/lsp: run gopls as long as there is one folder with a file URI d1954cc8
* internal/lsp/regtest: reproduce edge cases when creating new files c3046009
* internal/span: remove check for nil content bf6e7d25
* internal/lsp: add package completion suggestions cf97e2b3
* internal/lsp/lsprpc: make the LSP forwarder exit error clearer 80e1b039
* internal/lsp: use -json for module upgrades 6fbec87e
* unusedresult.go: Include context.With* functions 6e59e247
* internal/lsp: add a test that reproduces golang/go#37069 fd8950f6
* all: update dependencies in tools and gopls modules 5a2aa26b
* internal/lsp/cache: don't invalidate metadata for new invalid imports 5deb2631
* internal/lsp/fuzzy: fix return value on no match 6c983e67
* internal/lsp/source: remove dead code from completion 97019fc2
* internal/lsp/source: add some downranking for workspace symbols 3c805d23
* internal/lsp: refactor workspace Symbol method 88346e99
* internal/lsp: add completion suggestions for import statements 84ab5701
* internal/lsp/regtest: add benchmarks for IWL and completion 06cc1d0a
* lsp/general.go: change error messages to give more context 021d7c6f
* internal/lsp, go/packages: work-around invalid files in GOPATH mode debfaf60
* godoc: fix panic in Presentation.ServeFile 97606e32
* internal/lsp/source: do not allow extraction of an import spec 989ebae2
* internal/lsp: fix builds and tests for go1.12+ df83f4e7
* internal/lsp/regtest: add a workspace symbols benchmark 17fd2f27
* internal/lsp/source: sort references and implementations results b85e56c1
* internal/lsp: ignore period ('.') triggered completions in comments bc8aaaa2
* internal/lsp/source: use space character in markdown formatting f57a28cb
* internal/lsp/mod: handle nil pointer exception for empty go.mod files 28d6951f
* internal/lsp/source: fix completion on final line of a document e0bf2294
* internal/lsp: use prefix matcher with comment completion b72e8bb6
* internal/analysisinternal: add anonymous structs & interfaces to b303f430
* internal/lsp: don't panic if there are no suggested fixes c024452a
* internal/lsp: do not log failed suggested fix for fillstruct to stderr 307de81b
* internal/lsp: improvements for command messages 1e23e48a
* internal/lsp/regtest: reproduce empty xtest bug 3366bbee
* internal/lsp: add ast fields to comment completion for declarations 3509cdc6
* internal/lsp/source: fix panic in formatZeroValue for invalid type daa65388
* internal/lsp/source: fix composite literal type name completion 74543c40
* internal/lsp/lsprpc: quiet the handshaker during regtests c886c0b6
* internal/lsp: consolidate progress reporting ed71c572
* internal/lsp/source: improve completion of printf operands 9ac8e33b

### CODE REVIEW

File: testdriver/testdriver.go [hunk 1]
```diff
@@ -53,7 +53,7 @@ var (
 func init() {
 	v := os.Getenv(testsetup.EnvErrLogMatchWait)
 	if v == "" {
-		v = "30s"
+		v = "25s"
```
Comment (myitcv) [inline testdriver/testdriver.go:56]: I think we should probably vary this via a workflow env var for now.

**Review by myitcv — APPROVED** [review]
LGTM with one comment

File: testdriver/testdriver.go (hunk 1 above)
Comment (leitzler) [reply testdriver/testdriver.go]: Reverted and moved to the docker based tests
